ui/raidboss/emulator/data/CombatantState.ts

```diff
@@ -0,0 +1,88 @@
+export interface ICombatantState {
+  posX?: number;
+  posY?: number;
+  posZ?: number;
+  heading?: number;
+  targetable?: boolean;
+  HP?: number;
+  maxHP?: number;
+  MP?: number;
+  maxMP?: number;
+}
+
+// Member names taken from OverlayPlugin's MiniParse.cs
+// Types taken from FFXIV parser plugin
+export interface IPluginState {
+  CurrentWorldID?: number;
+  WorldID?: number;
+  WorldName?: string;
+  BNpcID?: number;
+  BNpcNameID?: number;
+  PartyType?: number;
+  ID?: number;
+  OwnerID?: number;
+  type?: number;
+  Job?: number;
+  Level?: number;
+  Name?: string;
+  CurrentHP: number;
+  MaxHP: number;
+  CurrentMP: number;
+  MaxMP: number;
+  PosX: number;
+  PosY: number;
+  PosZ: number;
+  Heading: number;
+}
+
+export default class CombatantState implements ICombatantState {
+  posX: number;
+  posY: number;
+  posZ: number;
+  heading: number;
+  targetable: boolean;
+  HP: number;
+  maxHP: number;
+  MP: number;
+  maxMP: number;
```

Comment:
```suggestion
  hp: number;
  maxHp: number;
  mp: number;
  maxMp: number;
```

Sorry again about the lack of lint rule.

Reply:
Had to update some other spots in code to reflect this change.